### ISSUE TITLE
Build Failure: ignore_sr_snapshots.gradle' as it does not exist.

### ISSUE THREAD

**Issue by masterwok** [DataDog/dd-sdk-flutter#326]
**Describe what happened**
Unable to build using latest version `1.2.0`. Receiving the following error:

```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/jt/.pub-cache/hosted/pub.dev/datadog_flutter_plugin-1.2.0/android/build.gradle' line: 70

* What went wrong:
A problem occurred evaluating project ':datadog_flutter_plugin'.
> Could not read script '/Users/jt/.pub-cache/hosted/ignore_sr_snapshots.gradle' as it does not exist.
```

**Steps to reproduce the issue:**

Building project after upgrading to `1.2.0`.

**Describe what you expected:**

The project to build successfully.

**Additional context**
 - Dart/Flutter version: 3.7
 - Android/iOS OS version: Android API 31
 - Device Model: Emulator
 - Datadog SDK version: 1.2.0
 - Versions of any other relevant dependencies: n/a

**Comment by masterwok** [comment]
I tried clearing the pub-cache manually using `flutter pub cache clean` but this did not resolve the issue.

**Comment by fuzzybinary** [comment]
Hi @masterwok - Thanks for the report. I'll get this fixed tonight.

**Comment by masterwok** [comment]
Thank you @fuzzybinary !

**Comment by fuzzybinary** [comment]
@masterwok Should be fixed with 1.2.1.
